Automattic/jetpack
# AI Assistant: Add disclaimer to image generation input

Fixes #

## Proposed changes:

This is an attempt to address feedback, as some people get frustrated with text errors that are beyond the capability of current models and maybe they are expecting it to work perfectly.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to an image generation modal (logo, featured or general image)
* Check that the disclaimer is there
* Check that the links work as expected

You can also run Storybook locally with `( cd projects/js-packages/storybook && pnpm run storybook:dev --no-open --debug-webpack )` and visiting `http://localhost:50240/?path=/docs/js-packages-ai-client-aimodalfooter--docs`





![Screenshot from 2024-12-03 20-00-22](https://github.com/user-attachments/assets/a2ad61ea-98a1-49f2-ab00-09fb2b8c0817)
![Screenshot from 2024-12-03 19-58-59](https://github.com/user-attachments/assets/966195b3-e715-4dfd-82dd-c7f1d5e0adb9)
